### ISSUE TITLE
Add wallpaper preview in shape size and appearance

### ISSUE THREAD

**Issue by Seandylan123** [patzly/doodle-android#77]
You can see a preview of the live wallpaper about how it looks to see the exact size you want or matches with your device

**Comment by patzly** [comment]
As I said, you have to understand that there simply is no correct scale for every device, it fully depends on the device's display properties. But a preview in this section is overall a good idea because of the coloring options.

**Comment by Seandylan123** [comment]
> As I said, you have to understand that there simply is no correct scale for every device, it fully depends on the device's display properties. But a preview in this section is overall a good idea because of the coloring options.

So is it plan to be added?

**Comment by patzly** [comment]
Yeah maybe, but it doesn't have a high priority because the user can go to the home screen to see the wallpaper and whether it has the right scale and appearance.

**Comment by Seandylan123** [comment]
> Yeah maybe, but it doesn't have a high priority because the user can go to the home screen to see the wallpaper and whether it has the right scale and appearance.

There are users (including me) that want this but you should probably only put it in the appearance because as you said it is a good idea because of the coloring options
